<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContestOptDialog</class>
 <widget class="QDialog" name="ContestOptDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>683</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Contest Options</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldGrowthPolicy::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="sentExch1Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exch1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sentExch2Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exch2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sentExch3Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exch3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="sentExch4Label">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Exch4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sent exchange:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEditExch1">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>First part of sent exchange. RST and Qso # are handled automatically.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditExch2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Second part of sent exchange. RST and Qso # are handled automatically.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditExch3">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Third part of sent exchange. RST and Qso # are handled automatically.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditExch4">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Fourth part of sent exchange. RST and Qso # are handled automatically.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="warningLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="MultiModeCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Stations can be worked for points on each mode</string>
       </property>
       <property name="text">
        <string>Multimode</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="SprintCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Special logic for Sprint QSY rule. If enabled, alt-D will be disabled.</string>
       </property>
       <property name="text">
        <string>Sprint Mode</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QCheckBox" name="MasterCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Supercheck Partial</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="MasterLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Filename of master call database, default location /usr/local/share/so2sdr.</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QCheckBox" name="HistoryCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Enable Call History for exchange Pre-fill</string>
       </property>
       <property name="text">
        <string>Call History</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="HistoryLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Exchange history database file.</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="QCheckBox" name="HistoryUpdateCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Append Call History on-the-fly during contest</string>
       </property>
       <property name="text">
        <string>Append Call History</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dupe Behavior</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="dupesComboBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>CQ: no dupes; S&amp;P: no dupes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CQ: work dupes; S&amp;P: no dupes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>no dupe checking</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
     <item row="15" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="16" column="0">
      <widget class="QCheckBox" name="OffTimeCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Enable tracking of off times.</string>
       </property>
       <property name="text">
        <string>Off Times</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Contest Start</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QDateTimeEdit" name="startDateTimeEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Start date/time of contest</string>
       </property>
       <property name="displayFormat">
        <string>M/d/yy hh:mm </string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::TimeSpec::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Contest End</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QDateTimeEdit" name="endDateTimeEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Last possible qso time.</string>
       </property>
       <property name="displayFormat">
        <string>M/d/yy hh:mm</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::TimeSpec::UTC</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Minimum duration</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLineEdit" name="offMinimumLineEdit">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Minimum length of off time in minutes</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="MultsByModeCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Mults by Mode</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="ShowMultsCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Show mults worked in &quot;Mults&quot; box</string>
       </property>
       <property name="text">
        <string>Show Mults</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="MultsByBandCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Multipliers are per-band</string>
       </property>
       <property name="text">
        <string>Mults By Band</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="ShowModeCheckBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Show mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="contest_dialog_buttons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>contest_dialog_buttons</sender>
   <signal>accepted()</signal>
   <receiver>ContestOptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contest_dialog_buttons</sender>
   <signal>rejected()</signal>
   <receiver>ContestOptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
